echopype/echodata/combine.py

```diff
@@ -13,7 +16,7 @@
 logger = _init_logger(__name__)
 
 
-def check_zarr_path(zarr_path: str, storage_options: Optional[dict]) -> str:
+def check_zarr_path(zarr_path: str, storage_options: Optional[dict], overwrite: bool) -> str:
```

Comment:
```suggestion
def check_zarr_path(zarr_path: str, storage_options: Dict = {}, overwrite: bool = False) -> str:
```

Few fixes here, probably good to set a default for `overwrite`. Also, seem like you did spread `storage_options` in line 68, there should be a default empty dictionary. `validate_output_path` doesn't take optional `storage_options`, so this shouldn't be optional as in `None` is able to be passed either. Whenever you have `Optional`, the default value should be `None` otherwise, that argument is required.